### PR TITLE
🔨  Implement script to deploy the contracts

### LINKED ISSUE #119: feat: deployment scripts of the unruggable contracts
Create deployment scripts using the method of your choice - either StarknetJS or Starknet-Foundry (I am not sure SNFoundry is ready for this ATM, but this can be a good exploration). 

These scripts will:
- Deploy the `TokenLocker` contract with a `min_lock_time` set to 6 months (`15_721_200` seconds)
- Declare the `UnruggableMemecoin` contract to get its class hash
- Deploy the `Factory` contract with:
  -  the class hash of the unruggable memecoin
  - the list of supported exchanges (type `(SupportedExchange, ContractAddress)`)

### PR DESCRIPTION
**Pull request by Bal7hazar**
Closes #119 

I made it in js, I think it will be easier to make evolutions. I would like to avoid bash since it is very relying on tools prompt outputs and the smaller change from these tools could break everything.
I did not made it in typescript since I though it was a bit overkill for a script, but if it comes to be bigger we may want to change this.

Notes:
- I had some issues to have an account and a RPC working with the latest starknet.js version.
- I had to use starkli to deploy a fresh new account (cannot make it works with braavos or argentx accounts).
- I had to use juno free RPCs for deploying the accounts with starkli
- I had to use infura RPC for deploying contracts through starknet.js

Workload ~ 6h

### CODE REVIEW

**Comment by Bal7hazar** [comment]
@enitrat ready for review

**Comment by enitrat** [comment]
> I had to use starkli to deploy a fresh new account (cannot make it works with braavos or argentx accounts).

just fyi - it should work with Braavos accounts, but you need to instanciate them in StarknetJS with the "0" version

File: contracts/scripts/libs/exchange.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const EXCHANGES = {
+  mainnet: [
+    {
+      name: "JediSwap",
+      contract_address: "",
```
Comment (enitrat) [inline contracts/scripts/libs/exchange.js:5]: can you fill this value?

File: contracts/scripts/libs/exchange.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      name: "JediSwap",
+      contract_address:
+        "0x02bcc885342ebbcbcd170ae6cafa8a4bed22bb993479f49806e72d96af94c965",
+    },
+  ],
+  sepolia: [
+    {
+      name: "JediSwap",
+      contract_address:
+        "0x02bcc885342ebbcbcd170ae6cafa8a4bed22bb993479f49806e72d96af94c965",
```
Comment (enitrat) [inline contracts/scripts/libs/exchange.js:19]: same as goerli? If it doesnt exist I think we should delete

**Review by enitrat — COMMENTED** [review]
Excellent work 💯

**Comment by Bal7hazar** [comment]
Done, thank you

**Pull request merged**